RELEASE.md

```diff
@@ -1,6 +1,7 @@
 # Upcoming Release 0.19.4
 
 ## Major features and improvements
+* Kedro commands now work from any subdirectory within a Kedro project.
```

Comment:
`_is_project` and `_find_kedro_project` are private methods, but it might be worth mentioning in the release notes section "Breaking changes to the API" that these were moved to `kedro.utils`, just in case people are using them.

Reply:
Updated!